examples/roitberg/pyanitools.py

```diff
@@ -0,0 +1,126 @@
+# Written by Roman Zubatyuk and Justin S. Smith
+import h5py
+import numpy as np
+import platform
+import os
+
+PY_VERSION = int(platform.python_version().split('.')[0]) > 3
+
+
+class datapacker(object):
```

Comment:
I'm more concerned about forking this code when release more datasets down the road and modify this. Do we want to contribute this change back upstream?